cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -264,3 +266,26 @@ def test_reports_delete_saved_report(appliance, request):
     view.configuration.item_select(
         item='Delete selected Saved Reports', handle_alert=True)
     assert not report.exists
+
+
+def test_create_custom_report_schedule(
+    appliance, request, custom_report_values, schedule_data
+):
+    """This test case creates a schedule for custom reports and tests if it was created
+    successfully.
+    """
+    custom_report = appliance.collections.reports.create(**custom_report_values)
+    schedule_data["filter"] = (
+        "My Company (All Groups)",
+        "Custom",
+        custom_report.menu_name,
+    )
+    try:
+        custom_report_schedule = appliance.collections.schedules.create(**schedule_data)
+    except (MoveTargetOutOfBoundsException, NoSuchElementException):
+        # Temporary workaround - running the `create` method first time raises exceptions,
```

Comment:
@valaparthvi I don't think we should be needing to try this twice, it should work the first time. Please spend some more time debugging what is happening. This may be a BZ.

Moving the PR back to WIPTEST